compiler/rustc_middle/src/query/mod.rs

```diff
@@ -1469,6 +1481,7 @@ rustc_queries! {
     /// For `struct Foo<'a, T: 'a, U>`, this would instead be `['a, 'static]`.
     query object_lifetime_defaults_map(_: LocalDefId)
         -> Option<Vec<ObjectLifetimeDefault>> {
```

Comment:
Can this be `Option<&'tcx [ObjectLifetimeDefault]>` instead of `&'tcx Option<Vec<ObjectLifetimeDefault>>`?